ql/src/test/results/clientpositive/llap/views_explain_ddl.q.out

```diff
@@ -305,7 +305,7 @@ TBLPROPERTIES (
 ALTER TABLE db1.table2_n13 UPDATE STATISTICS SET('numRows'='0','rawDataSize'='0' );
 ALTER TABLE db1.table1_n19 UPDATE STATISTICS SET('numRows'='0','rawDataSize'='0' );
 
-CREATE VIEW `db1`.`v3_n3` AS SELECT `t1`.`key`, `t1`.`value`, `t2`.`key` `k` FROM `db1`.`table1_n19` `t1` JOIN `db1`.`table2_n13` `t2` ON `t1`.`key` = `t2`.`key`;
+CREATE VIEW `db1`.`v3_n3` AS SELECT `t1`.`key`, `t1`.`value`, `t2`.`key` `k` FROM `db1`.`table1_n19` `t1` JOIN `db1`.`table2_n13` `t2` ON t1.key = t2.key;
```

Comment:
View expanded text changed: quotation removed from table and column names in join condition: 
```
t1.key = t2.key
```
should remain 
```
`t1`.`key` = `t2`.`key`
```

Reply:
Makes sense.  Didn't know what the unparsed translator does until now, thanks!